ext/standard/math.c

```diff
@@ -1023,6 +1023,17 @@ PHP_FUNCTION(dechex)
 }
 /* }}} */
 
+ZEND_FRAMELESS_FUNCTION(dechex, 1)
+{
+	zend_long arg;
+
+	Z_FLF_PARAM_LONG(1, arg);
+
+	RETVAL_STR(_php_math_longtobase_pwr2(arg, 4));
+
+flf_clean:;
```

Comment:
Label may be removed

Reply:
`Z_FLF_PARAM_LONG` may fail, so I don't think I can.

Reply:
Ah ok, my mistake :-)